source/cellular/known-working-3g-4g-modems.rst

```diff
@@ -91,6 +91,10 @@ pfSense® software.
 -  Sierra Wireless MC7354
 -  Sierra Wireless MC7355
 -  Sierra Wireless MC7710
+
+   - May require usb-comp or ID change, see
+     `this topic <https://forum.netgate.com/post/556751>`__
```

Comment:
We try to avoid using links like "click here" or "this topic" since they don't provide much useful context to the reader (See https://docs.netgate.com/pfsense/en/latest/references/style-guide.html#what-to-avoid ).

You could make the entire text a link: `` `May require usb-comp or ID change <https://forum.netgate.com/post/556751>`__ ``

Or add the link without using link text: `` See https://forum.netgate.com/post/556751 for more information. ``

Reply:
ok, fixed